esignet-core/src/main/java/io/mosip/esignet/core/util/CaptchaHelper.java

```diff
@@ -29,11 +35,33 @@ public class CaptchaHelper {
     private String moduleName;
     private String validatorUrl;
 
+    private List<String> captchaRequired;
+
     public CaptchaHelper(RestTemplate restTemplate, String validatorUrl, String moduleName) {
         this.restTemplate = restTemplate;
         this.validatorUrl = validatorUrl;
         this.moduleName = moduleName;
     }
```

Comment:
Can we remove this old constructor? so it will not confuse in the future.

Reply:
Removed the old constructor